uwblueprint/planet-read
# Setting Up GitHub Action to deploy to Firebase Hosting

## Notion ticket link
https://www.notion.so/uwblueprintexecs/Create-development-environment-2d4b16cf1a834269ba86ec183d058575
[Create Development Environment](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* Followed the steps outlined on https://firebase.google.com/docs/hosting/github-integration to create the necessary files to auto deploy to firebase every time a new PR is merged. 



## Steps to test
1. Make new pull request and see if there is an option to preview the changes before merge 



## What should reviewers focus on?
* 


## Checklist
